tf-modules/cumulus/ecs_cluster_instance_autoscaling_cf_template.yml.tmpl

```diff
@@ -26,20 +26,19 @@ Resources:
           --==BOUNDARY==
           Content-Type: text/cloud-boothook; charset="us-ascii"
 
-%{ if docker_storage_driver == "devicemapper" ~}
           vgcreate docker /dev/xvdcz
 
           lvcreate -n docker-data -L${docker_volume_create_size}G docker
 
-          sed -i '/^\s*DOCKER_STORAGE_OPTIONS=/d' /etc/sysconfig/docker-storage
-          echo 'DOCKER_STORAGE_OPTIONS="--storage-driver devicemapper --storage-opt dm.datadev=/dev/docker/docker-data"' >> /etc/sysconfig/docker-storage
+          mkfs.xfs /dev/docker/docker-data
```

Comment:
Does this approach work for other reasonable configuration options?     Or more pointedly, we have a configuration option that allows users to set the driver type, is that actually meaningful still?

Reply:
Following up, this will probably break for non-overlay options like btrfs, but that might have already not worked given how I'm reading this config 🤔

Reply:
only use `overlay2` as storage driver